components/Extensions/samples/ShadowAnimations.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Shadow animations
+author: michael-hawker
+description: Animating Attached Shadows
+keywords: shadow, shadows, dropshadow, dropshadowpanel, attachedshadow, attacheddropshadow, attachedcardshadow
+dev_langs:
+  - csharp
+category: Animations
+subcategory: Shadows
+discussion-id: 0
+issue-id: 0
+icon: Assets/ShadowAnimation.png
+---
+
+Either type of Attached Shadow can be easily animated using the Toolkit's [`AnimationSet`](../animations/AnimationSet.md) api. These provide an easy XAML based way to animate a wide variety of elements, including a variety of shadow properties. They can also be animated with any other composition animation technique in code-behind as well using either the [`AnimationBuilder`](../animations/AnimationBuilder.md) or built-in composition animations.
+
+> **Platform APIs:** [`BlurRadiusDropShadowAnimation`](/dotnet/api/microsoft.toolkit.uwp.ui.animations.blurradiusdropshadowanimation), [`ColorDropShadowAnimation`](/dotnet/api/microsoft.toolkit.uwp.ui.animations.colordropshadowanimation), [`OffsetDropShadowAnimation`](/dotnet/api/microsoft.toolkit.uwp.ui.animations.offsetdropshadowanimation), [`OpacityDropShadowAnimation`](/dotnet/api/microsoft.toolkit.uwp.ui.animations.opacitydropshadowanimation)
```

Comment:
We should think about these sections as none of these links are going to work for now... 🤔

Unless we replace them with the `CommunityToolkit.WinUI.Animations...` equivalent's but those pages won't be up for a bit still anyway

Reply:
Perhaps we just notify the user that updated docs are coming soon?

Reply:
Moved here - https://github.com/CommunityToolkit/Windows/pull/213